.travis.yml

```diff
@@ -1,23 +1,20 @@
 language: python
 
-python:
-    - "2.7"
-
 virtualenv:
   system_site_packages: true
 
 env:
-  - NEURODEBIAN=0
-  - NEURODEBIAN=1
-
-before_install:
-    - wget -O- http://neuro.debian.net/lists/precise.us-nh.libre | sudo tee /etc/apt/sources.list.d/neurodebian.sources.list
-    - sudo apt-key adv --recv-keys --keyserver pgp.mit.edu 2649A5A9
-    - if [ $NEURODEBIAN -eq 1 ]; then sudo apt-get update -qq; fi
-    - sudo apt-get install -qq python-scipy python-nose python-nibabel python-sklearn
+  matrix:
+    - DISTRIB="ubuntu" PYTHON_VERSION="2.7"
+      COVERAGE="false"
+    - DISTRIB="neurodebian" PYTHON_VERSION="2.7"
+      COVERAGE="false"
```

Comment:
I will remove these and add an issue for setting up `coversall`.  It's along the lines of things I'm able and happy to contribute.